src/Sync.js

```diff
@@ -206,10 +207,13 @@ class Sync {
     // Overwrite the files to the target directories
     await this.writeFiles(splicedFiles);
     // Delete the sync_repos directory
+
     await this.cleanUp();
     this.progress.updateOperation("done");
     this.progress.stop();
     this.logger.info("snipsync operation complete");
+    this.reportUnusedRepos(usedRepos);
+    console.log("done");
```

Comment:
Is this needed? The logger statement above seems to cover this. I wonder if you move the previous line above the logger, that's enough. If you disagree, I think I'd use this.logger instead of console.log.